perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -38,9 +38,17 @@ sub disambiguate_location {
         $town = ', BR6 0PL';
     }
     $town .= ', BR5' if $string =~ /^meadway/i;
+    $town .= ', BR5' if $string =~ /^mill\s+brook\s+r(?:oa)?d$/i;
+    $town .= ', BR5' if $string =~ /^kent\s+r(?:oa)?d$/i;
+    $town .= ', BR5' if $string =~ /^the\s+landway$/i;
+    $town .= ', BR5' if $string =~ /^mountfield\s+way$/i;
+    $town .= ', BR5 3' if $string =~ /^star\s+lane$/i;
+    $town .= ', BR5 4AX' if $string =~ /^high\s+st(reet)?(,)?\s+st\s+mary\s+cray$/i;
```

Comment:
Perhaps space optional after the comma? And optional dot after the `st` I guess.